modules/nf-core/magus/guidetree/main.nf

```diff
@@ -0,0 +1,48 @@
+process MAGUS_GUIDETREE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::magus-msa=0.1.2"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/magus-msa:0.1.2--pyhdfd78af_0 ':
+        'biocontainers/magus-msa:0.1.2--pyhdfd78af_0 ' }"
+
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    tuple val(meta), path("*.tree"), emit: tree
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    magus \\
+        -np $task.cpus \\
+        -i $fasta \\
+        -o ${prefix}.tree \\
+        --onlyguidetree TRUE \\
```

Comment:
I don't know what the tool does, what other options could this be?

Reply:
The parameters mostly have to do with tweaking the way the multiple seuqence alignment is constructed.
They are documented in the help message (s. also https://github.com/vlasmirnov/MAGUS/blob/master/magus.py#L43 )

Reply:
But you are hardcoding this one to`--onlyguidetree TRUE`, could someone actually want the other outputs?

Reply:
The other output would be the alignment, which can be obtained by the sister module magus/align.
Theoretically, one could also pass `--onlyguidetree FALSE` to args which would overwrite this setting, but I don't see much point in doing that rather than calling the correct module.